outputs.tf

```diff
@@ -41,10 +41,20 @@ output "service_accounts" {
 
 output "emails" {
   description = "Service account emails."
-  value       = google_service_account.service_accounts[*].email
+  value       = zipmap(var.names, google_service_account.service_accounts[*].email)
```

Comment:
Maybe better ` value       = zipmap(var.names, google_service_account.service_accounts.*.email)`

Reply:
I kind of prefer the new splat syntax to the legacy one. What's your motivtion for using legacy?